examples/kubernetes/windows/csi-driver.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: storage.k8s.io/v1beta1
```

Comment:
Please include this file with the driver-deployment spec, instead of with user-level examples (pod, pvc, etc)